app/src/main/res/values/colors_migrating.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```

Comment:
I suggest adding a top-level comment explaining what colors should go in this file. We also ought to add a CODEOWNERS rule and make you & Rajat the codeowners specifically for this file so that people don't add to it (otherwise it'll make the migration harder).

Reply:
PTAL.